src/services/tables/builders/PriceEstimatesTableBuilder.js

```diff
@@ -6,8 +6,33 @@ import Table from 'cli-table2';
 
 import Utilities from '../../../Utilities';
 
+var emojiCarType = emoji.get('oncoming_automobile');
```

Comment:
@sjs7007 thank you for doing this (and sorry about the wait)!

I wonder, and maybe this is total overkill, but do you think some sort of "symbol" service might make this logic a little cleaner? This service would abstract away the `process.platform` call.

Let me know your thoughts! If you're busy, I'm more than happy to do a first pass on what this service might look like.